### DISCUSSION TITLE
Can xstyled be used with SolidJS?

### DISCUSSION THREAD

**Discussion by thedanchez** [styled-components/xstyled#393] [Q&A]
Hello, I'm trying to make a custom component library in SolidJS and I would like for my components to have a style-props like interface on them. Is there an agnostic part of xstyled that allows me to add all the possible style-based props to my components? Or does this only work for React?

**Comment by jguddas** [comment]
Nope, you could use something like stitches but I wouldn't recommend it.
Here @fabien-ml explains some of the downsides https://github.com/hope-ui/hope-ui/issues/187#issuecomment-1385235338
